src/draw/nema_gfx/lv_draw_nema_gfx_hal.c

```diff
@@ -12,11 +12,13 @@
 #include "../../lv_conf_internal.h"
 #if LV_USE_NEMA_GFX
 
+#if LV_USE_PROVIDED_NEMA_GFX_HAL_FOR_STM32
```

Comment:
Let's rename this file to `lv_draw_nema_gfx_stm32_hal.c`.
Probably we will have more mutually exclusive HAL's supported later so we can create a structure similar to `LV_USE_OS`. E.g.
```c
#define LV_USE_NEMA_HAL LV_NEMA_HAL_STM32/CUSTOM
```

Reply:
Will `#define LV_USE_NEMA_HAL LV_NEMA_HAL_STM32/CUSTOM` be added too?

Reply:
As only one HAL can be selected at time, make as described above.

Reply:
> Will `#define LV_USE_NEMA_HAL LV_NEMA_HAL_STM32/CUSTOM` be added too?

> As only one HAL can be selected at time, make it as described above.

Got it. Please see my latest commit.